### ISSUE TITLE
Additional mutations: Cephalopod - Defensive Ink Spray/Blind

### ISSUE THREAD

**Issue by OldFriendEr** [CleverRaven/Cataclysm-DDA#38692]
# Is your feature request related to a problem? Please describe.
As I was playing CDDA and reading the issues I noticed that there could be more mutations in specific trees (rather having more trees) and that some trees have way more mutations that others. I decided to role-play a cephalopod and I noticed that it could have more unique mutations. I realized that a cephalopod  mutant currently lacks a signature mutation: ink/spray (its not that useful or essential, but I think would really add some character for this mutant).

> Fishermen sometimes call cephalopods "inkfish,"

# Describe the solution you'd like

A mutation that has a special attack: ink spray that blinds a valid target and deal no or a single point of damage to the eyes. 
Ideally I would like that it would only activate while counterattacking/blocking or getting attacked while moving, since the nature of ink spray is **defensive** rather than offensive.

# Describe alternatives you've considered
not implementing it

# Additional context

Right now my goal is to enrich the cephalopod mutation tree and this is my first attempt.
I'm actually up to implement this mutation myself. ( i'll poke around the discord for that.)

The actual reason why I'm submitting a issue because I wish to know, if you think that my goal (enriching a specific mutation tree with more mutations) is in line with the CDDA and will not be rejected after its done.

This mutation could also be shared with slime, be cause why not (slime spray instead of ink).

I'm looking forward for your feedback

# Balance 

* The ink sacs should be located somewhere on front torso section, so it only activates when not wearing anything on torso, while being hit on any body part. 
* Is intended to compliment the shell mutation ( cant wear anything on torso)
* The effect is two turn blind.
* Haven't decided on a cooldown or a chance yet.
* I'd say that mutation should be passive (like an adrenaline rush) otherwise it would be quite cumbersome to fire it on demand

**Comment by Amoebka** [comment]
There is already a mutation that blinds attackers upon receiving damage - Urticating Hairs from the arachnid line. You can copy the functionality and refluff the description if you wish.

A much better (and much harder) way of doing it is allowing mutations to cast spells upon activation. This way all the rich functionality of spells can be used to create various active mutations with cooldowns.

**Comment by KorGgenT** [comment]
you know i don't think it would be _that_ hard to make mutations cast spells upon activation. it'll require some C++, but most mutations do.

**Comment by OldFriendEr** [comment]
Big thanks for @Amoebka for the Urticating Hairs mutation, I could already finish this mutation just by this example alone.

I poked around JSON files of base game and magiclysm to find any mutations that casted spells upon activation, but failed to locate any. I also failed to locate any player spell effects in the base game so I'm not sure in which files I should add that spell, for the mutation to cast.
